Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ButtonGroup.h

```diff
@@ -31,6 +31,7 @@ namespace AzToolsFramework::ViewportUi::Internal
         ViewportUiElementId GetViewportUiElementId() const;
 
         ButtonId AddButton(const AZStd::string& icon, const AZStd::string& name = AZStd::string());
+        void RemoveButton(ButtonId buttonId);
```

Comment:
suggestion: It might be useful (especially for testing too) to have `RemoveButton` return a boolean to indicate if the button was successfully removed or not. These types of things are nearly always useful to include and can help test class functionality.